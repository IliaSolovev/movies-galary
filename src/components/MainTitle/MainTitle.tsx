import React from 'react';
import s from './mainTitle.module.scss';

const MainTitle: React.FC = () => (
  <div className={s.title}>netflixroulette</div>
);

export default MainTitle;
