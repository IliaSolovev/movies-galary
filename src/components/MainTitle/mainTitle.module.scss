@import "../../styles/variables";
.title{
  font-family: $mainFont;
  font-size: 1.2em;
  font-weight: bold;
  color: $redColor;
}
