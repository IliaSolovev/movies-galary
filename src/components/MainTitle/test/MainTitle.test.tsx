import React from "react";
import {shallow} from "enzyme";
import MainTitle from "../MainTitle";
it('should render main title', () => {
    const component = shallow(<MainTitle/>);
    expect(component).toMatchSnapshot();
});