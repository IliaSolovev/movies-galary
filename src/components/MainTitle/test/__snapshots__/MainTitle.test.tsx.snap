// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render main title 1`] = `
<div
  className="title"
>
  netflixroulette
</div>
`;
